code/modules/events/space_dragon.dm

```diff
@@ -0,0 +1,45 @@
+/datum/round_event_control/space_dragon
+	name = "Spawn Space Dragon"
+	typepath = /datum/round_event/ghost_role/space_dragon
+	weight = 8
+	max_occurrences = 1
+	min_players = 20
```

Comment:
The thing is with blob, it can lose relatively easy with only 3 people fighting it.  35 pop is still too high for blob, especially now adays when we have even less.

Reply:
> The thing is with blob, it can lose relatively easy with only 3 people fighting it. 35 pop is still too high for blob, especially now adays when we have even less.

account for the rate of coomers per players
twenty players might have at most two competents and likely no security